packages/nvidia_nat_agno/tests/test_llm_agno.py

```
@@ -137,15 +131,24 @@ async def test_nim_agno_without_api_key(self, mock_nvidia, nim_config, mock_buil
                 mock_nvidia.assert_called_once()
                 assert nvidia_instance == mock_nvidia.return_value
 
+    @pytest.mark.parametrize("verify_ssl", [True, False], ids=["verify_ssl_true", "verify_ssl_false"])
+    @patch("agno.models.nvidia.Nvidia")
+    async def test_nim_agno_verify_ssl_passed_to_client(self,
+                                                        mock_nvidia,
```

Ruff (0.15.5)

[warning] Unused method argument: `mock_nvidia`

(ARG002)